mne/decoding/transformer.py

```diff
@@ -0,0 +1,455 @@
+            The data.
+        y : None
+            Not used.
+
+        Returns
+        -------
+        X : array, shape=(n_epochs, n_channels, n_times)
+            The data after filtering
+        """
+        if not isinstance(epochs_data, np.ndarray):
+            raise ValueError("epochs_data should be of type ndarray (got %s)."
+                             % type(epochs_data))
+
+        epochs_data = np.atleast_3d(epochs_data)
+
+        if self.l_freq is None and self.h_freq is not None:
+            epochs_data = \
+                low_pass_filter(epochs_data, self.info['sfreq'], self.h_freq,
+                                filter_length=self.filter_length,
+                                trans_bandwidth=self.l_trans_bandwidth,
+                                method=self.method, iir_params=self.iir_params,
+                                picks=self.picks, n_jobs=self.n_jobs,
+                                copy=False, verbose=False)
+
+        if self.l_freq is not None and self.h_freq is None:
+            epochs_data = \
+                high_pass_filter(epochs_data, self.info['sfreq'], self.l_freq,
+                                 filter_length=self.filter_length,
+                                 trans_bandwidth=self.h_trans_bandwidth,
+                                 method=self.method,
+                                 iir_params=self.iir_params,
+                                 picks=self.picks, n_jobs=self.n_jobs,
+                                 copy=False, verbose=False)
+
+        if self.l_freq is not None and self.h_freq is not None:
+            if self.l_freq < self.h_freq:
+                epochs_data = \
+                    band_pass_filter(epochs_data, self.info['sfreq'],
+                                     self.l_freq, self.h_freq,
+                                     filter_length=self.filter_length,
+                                     l_trans_bandwidth=self.l_trans_bandwidth,
+                                     h_trans_bandwidth=self.h_trans_bandwidth,
+                                     method=self.method,
+                                     iir_params=self.iir_params,
+                                     picks=self.picks, n_jobs=self.n_jobs,
+                                     copy=False, verbose=False)
+            else:
+                epochs_data = \
+                    band_stop_filter(epochs_data, self.info['sfreq'],
+                                     self.h_freq, self.l_freq,
+                                     filter_length=self.filter_length,
+                                     l_trans_bandwidth=self.h_trans_bandwidth,
+                                     h_trans_bandwidth=self.l_trans_bandwidth,
+                                     method=self.method,
+                                     iir_params=self.iir_params,
+                                     picks=self.picks, n_jobs=self.n_jobs,
+                                     copy=False, verbose=False)
```

Comment:
this block above is a duplicate of what's in the raw.filter method. It should be refactored.